storage/Devices/BcacheImpl.cc

```diff
@@ -415,7 +502,19 @@ namespace storage
 
 	SystemCmd cmd(cmd_line, SystemCmd::DoThrow);
 
-	sleep(1);
+	sleep(1); // TODO: is it enough? check with Coly how to recognize it
```

Comment:
WTF, please add a comment *why* a sleep is needed.

Reply:
OK, reason is that all bcache sysfs operations are async, so hard to say when it ends.

Reply:
WTF, then we must add a loop to wait for completion, or we will get super funny bugs, right?

Reply:
Yes, an in BcacheImpl.cc are several TODOs, one being "synchronization after actions".

Reply:
@mvidner in theory yes, but for same actions I need coly to say me which file have to appear and also some operation together with certain state of cache result in several hours or days till finish of operation ( detach one ). I can forward you notes from my conversation with Coly when he was in PRG.